src/OpenLoco/src/Ui/Windows/Vehicle.cpp

```diff
@@ -1270,7 +1270,7 @@ namespace OpenLoco::Ui::Windows::Vehicle
             {
                 if (itemIndex <= 0)
                 {
-                    BuildVehicle::open(self.number, 0);
+                    BuildVehicle::open(self.number, false);
```

Comment:
This is the one caller that uses entityId i'd prefer we just expose an entityId function rather than open(entityId, false)